packages/compass-editor/src/ace/theme.ts

```diff
@@ -1,12 +1,100 @@
 import { palette, spacing } from '@mongodb-js/compass-components';
 
+const mongodbAceThemeCssClass = 'ace-mongodb';
+
 // To trick vscode into formatting and autocompleting the string
 const css = String.raw;
 
 const mongodbAceThemeCssText = css`
+  .ace-mongodb {
```

Comment:
are those classes really global? or that "css" template does some magic? Can we make them namespaced to `mongodb-compass-*` otherwise?

Reply:
This is just a string of text that ace will insert in the style tag and so no transformation happens for the class name. I can change the prefix, but as this implementation is not changing current behavior and I have a bunch of PRs piling up, I'll merge this as-is and do a follow up with the change